### PR TITLE
docs: update zh-cn of install drivers page

### PR DESCRIPTION
**Pull request by jasonren0403**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

### Description
translate remaining EN content on install drivers page and change wording that may cause conflict

### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by alaahong** [comment]
Hi，
Thanks for your picking up the translation. It's a good start.
But can you do not change as below? Is there any necessary to replace all lines as yours even without revised?


image

**Review by alaahong — CHANGES_REQUESTED** [review]
Please do your changes based on existing content. You should keep compatibility if there was not exactly improvement on the translation. And meanwhile, you often merge several lines into one line as a long format (actually no change in rendered HTML) , even we mentioned you the contribution RULE. 

Otherwise, anyone also need change yours by any small point.

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md [hunk 1]
```diff
@@ -14,15 +14,13 @@ aliases: [
 通过WebDriver, 
 Selenium支持市场上所有主要浏览器, 
 如Chrome、Firefox、Internet Explorer、Edge和Safari. 
-WebDriver尽量使用浏览器内置的自动化支持
-来驱动浏览器.
+WebDriver尽量使用浏览器内置的自动化支持来驱动浏览器.
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]:
```suggestion
WebDriver尽量使用浏览器内置的自动化支持
来驱动浏览器.
```

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md [hunk 2]
```diff
@@ -14,15 +14,13 @@ aliases: [
...
 
-由于除Internet Explorer之外的所有驱动程序实现
-都是由浏览器供应商自己提供的, 
+由于除Internet Explorer之外的所有驱动程序实现都是由浏览器供应商自己提供的,
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]:
```suggestion
由于除Internet Explorer之外的所有驱动程序实现
都是由浏览器供应商自己提供的, 
```

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md [hunk 3]
```diff
@@ -14,15 +14,13 @@ aliases: [
...
+WebDriver尽量使用浏览器内置的自动化支持来驱动浏览器.
 
-由于除Internet Explorer之外的所有驱动程序实现
-都是由浏览器供应商自己提供的, 
+由于除Internet Explorer之外的所有驱动程序实现都是由浏览器供应商自己提供的, 
 因此标准Selenium发行版中不包括这些驱动程序. 
 本节介绍了使用不同浏览器的基本要求.
 
-在我们的[驱动程序配置]({{< ref "/documentation/webdriver/drivers/" >}}) 文档中
+在我们的[驱动程序配置]({{< ref "/documentation/webdriver/drivers/" >}}) 文档中，
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]:
```suggestion
在我们的[驱动程序配置]({{< ref "/documentation/webdriver/drivers/" >}}) 文档中
```

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md [hunk 4]
```diff
@@ -219,8 +218,7 @@ ChromeDriver was started successfully.
 
 与上面的选项3类似, 
 您需要手动下载驱动程序(有关链接, 请参阅[快速参考](#快速参考) 部分). 
-在代码中指定位置本身的优点是
-不需要指出系统上的环境变量, 
+在代码中指定位置本身的优点是不需要指出系统上的环境变量,
```
Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]:
```suggestion
在代码中指定位置本身的优点是
不需要指出系统上的环境变量, 
```

File: website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md [hunk 5]
```diff
@@ -34,29 +34,28 @@ WebDriver尽量使用浏览器内置的自动化支持
...
-Selenium Manager helps you to get a working environment to run Selenium out of the box. Beta 1
-of Selenium Manager will configure the drivers for Chrome, Firefox, and Edge if they are not 
-found on the `PATH`. No extra configuration is needed. Future releases of Selenium Manager 
-will eventually even download browsers if necessary.
+Selenium Manager可以帮助你获得一个运行Selenium的开箱即用的环境。
+如果在`PATH`中没有找到Chrome、Firefox和Edge的驱动，Selenium Manager的Beta 1版将为它们配置。
+不需要额外的配置。如果有必要，Selenium Manager的未来版本也会在必要时一同下载浏览器。
 
-Read more at the blog announcement for [Selenium Manager ](/blog/2022/introducing-selenium-manager/).
+阅读有关 [Selenium Manager](/blog/2022/introducing-selenium-manager/) 的文章。
```
Comment (alaahong) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]: more at the blog announcement？

Comment (jasonren0403) [inline website_and_docs/content/documentation/webdriver/getting_started/install_drivers.zh-cn.md]:
```suggestion
在这篇[公告](/blog/2022/introducing-selenium-manager/)中了解更多有关 Selenium Manager 的信息。
```

**Review by alaahong — APPROVED** [review]

**Comment by alaahong** [comment]
Thanks you @jasonren0403 !

**Pull request merged**